schema/DoctrineMigrations/Version20230110150114.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230110150114 extends AbstractMigration
```

Comment:
`use Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration;`

```suggestion
final class Version20230110150114 extends LoggableMigration
```